docs/learn/background/interop.md

```diff
@@ -45,7 +45,7 @@ Below, we'll refer to the _**I**nter**c**hain **S**tandards_, or ICS-x for short
 
 ## Tackling Interoperability Fragmentation
 
-Counterintuitively, the _transport layer_ is arguably the most crucial layer in the interoperability model. This layer creates a commitment to all messages sent and received from a chain, while also enforcing TAO logic. This commitment is known as a [_transport commitment_](../concepts/vibc/clients.md/#fork-handling-and-virtual-roots).
+Counterintuitively, the _transport layer_ is arguably the most crucial layer in the interoperability model. This layer creates a commitment to all messages sent and received from a chain, while also enforcing TAO logic. This commitment is known as a [_transport commitment_](../concepts/vibc/clients.md#fork-handling-and-virtual-roots).
```

Comment:
[Suggestion] Remove Counterintuitively here. Transport intuitively seems critical to interoperability, and because this is in the background users may not have any intuition.

Langauge could possibly be clarified here or extrapolated on. What does creating a commitment to all messages mean? Does it map the message being sent with the transport commitment or something different?